coreosBuilder.py

```diff
@@ -148,6 +149,7 @@ def build_step(cmd, step):
             sys.exit(-1)
 
         self._embed_ign(embed_src, dst)
+        os.chdir(cur_dir)
```

Comment:
The problem here is that this function has now a side effect on the process itself. That's a bit unexpected. Can we keep avoid calling `os.chdir` or at least cwd first and then chdir back? I know there are other places where we `chdir` but that makes `ensure_oc_binary` unhappy. I realize I'm asking to fix a general bug too with this comment.

Reply:
Not sure I understand - we now os.getcwd() at the ~begin of build, then chdir to fcos_dir (which was done before), then change back at the end to cwd. Before we were changing dir to fcos_dir.